src-cr/Containers/PlacesCollection.js

```diff
@@ -0,0 +1,50 @@
+import React from "react"
+
+import PlaceCard from "../Components/PlaceCard"
+
+import HeroImage from "../Components/HeroImage"
+
+
+class PlacesCollection extends React.Component{
+
+  render(){
+
+    const place = this.props.places.map(place =>
+
+      <PlaceCard key={place.id} place={place}/>)
+
+  // <HeroImage/>
+    return(
+
+     <div>
+       <div className="ui secondary pointing massive menu">
+         <a className="active grey item" href="/nycplaces">
+           NYC Places
+         </a>
+       </div>
+
+        <div className="ui container">
+          <HeroImage/>
+          <br/>
+          <select name="select" onChange={this.props.handleChange} className="ui fluid normal dropdown">
+            <option value="all">All</option>
```

Comment:
You can try out placing all this values on an array and mapping them.

```JS
const categories = ['sightseeing', 'discovering'];

// Then on your render method

<select>
  {categories.map(category => <option value={category}>upperFirst(category)</option>)}
</select>
```
https://lodash.com/docs/4.17.5#upperFirst